docs/paper/admin/reference/configuration/spigot-configuration.mdx

```diff
@@ -0,0 +1,563 @@
+        flying-monsters-every:
+          default: '200'
+          description: 'How often an inactive flying monster outside of range will be woken up, in ticks.
+            This is a minimum due to being limited to flying-monsters-max-per-tick.'
+        flying-monsters-for:
+          default: '100'
+          description: 'How long to wake an inactive flying monster up for, in ticks.'
+        flying-monsters-max-per-tick:
+          default: '8'
+          description: 'A limit of how many inactive flying monsters can be woken up on the same tick.'
+        monsters-every:
+          default: '400'
+          description: 'How often an inactive monster outside of range will be woken up, in ticks.
+            This is a minimum due to being limited to monsters-max-per-tick.'
+        monsters-for:
+          default: '100'
+          description: 'How long to wake an inactive monster up for, in ticks.'
+        monsters-max-per-tick:
+          default: '8'
+          description: 'A limit of how many inactive monsters can be woken up on the same tick.'
+        villagers-every:
+          default: '600'
+          description: 'How often an inactive villager outside of range will be woken up, in ticks.
+            This is a minimum due to being limited to villagers-max-per-tick'
+        villagers-for:
+          default: '100'
+          description: 'How long to wake an inactive villager up for, in ticks.'
+        villagers-max-per-tick:
+          default: '4'
+          description: 'A limit of how many inactive villagers can be woken up on the same tick.'
+      water:
+        default: '16'
+        description: 'The entity activation range for water mobs.'
+    entity-tracking-range:
+      animals:
+        default: '48'
+        description: 'Controls how far in blocks animals are tracked (sent to) the player.
+          This is scaled by entityBroadcastRangePercentage.'
+      display:
+        default: '128'
+        description: 'Controls how far in blocks display entities are tracked (sent to) the player.
+          This is scaled by entityBroadcastRangePercentage.'
+      misc:
+        default: '32'
+        description: 'Controls how far in blocks misc entities are tracked (sent to) the player.
+          This is scaled by entityBroadcastRangePercentage.'
+      monsters:
+        default: '48'
+        description: 'Controls how far in blocks monsters are tracked (sent to) the player.
+          This is scaled by entityBroadcastRangePercentage.'
+      other:
+        default: '64'
+        description: 'Controls how far in blocks other entities are tracked (sent to) the player.
+          This is scaled by entityBroadcastRangePercentage.'
+      players:
+        default: '48'
+        description: 'Controls how far in blocks players are tracked (sent to) the player.
+          This is scaled by entityBroadcastRangePercentage.'
+    growth:
```

Comment:
This section miss two things: the max efficient percent (how far the percent works if known) and the fact that setting this to zero will reset to 100% and will not prevent the grow. Also the message should be custom per plant

Reply:
was a pain, but think i got it